projects/valtimo/components/src/lib/components/multiselect-dropdown/multiselect-dropdown.component.html

```diff
@@ -0,0 +1,7 @@
+<ng-multiselect-dropdown
+        [placeholder]="placeHolder"
```

Comment:
Please correctly format file (too many tabs on the properties) and add copyright